resources/zib/zib-TreatmentDirective2.xml

```diff
@@ -0,0 +1,334 @@
+        <key value="zib-TreatmentDirective2-01" />
+        <severity value="error" />
+        <human value="`extension.additionalAdvanceDirective` may only be used for AdvanceDirective references in addition to first reference in `.source[x]`." />
+        <expression value="extension.where(url = 'http://nictiz.nl/fhir/StructureDefinition/ext-TreatmentDirective2.AdvanceDirective').empty() or source.exists()" />
+        <source value="http://nictiz.nl/fhir/StructureDefinition/zib-TreatmentDirective2" />
+      </constraint>
+      <constraint>
+        <key value="zib-TreatmentDirective2-02" />
+        <severity value="error" />
+        <human value="If the specificationOther extension is used, `.provision.type` MUST be absent and vice versa" />
+        <expression value="(extension.where(url = 'http://nictiz.nl/fhir/StructureDefinition/ext-TreatmentDirective2.SpecificitionOther').exists() and provision.type.exists()).not()" />
+        <source value="http://nictiz.nl/fhir/StructureDefinition/zib-TreatmentDirective2" />
+      </constraint>
+      <mapping>
+        <identity value="zib-treatmentdirective2-v1.0-2020EN" />
+        <map value="NL-CM:2.2.1" />
+        <comment value="TreatmentDirective" />
+      </mapping>
+    </element>
+    <element id="Consent.extension">
+      <path value="Consent.extension" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+      <min value="0" />
+    </element>
+    <element id="Consent.extension:comment">
+      <path value="Consent.extension" />
+      <sliceName value="comment" />
+      <min value="0" />
+      <max value="1" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-Comment" />
+      </type>
+    </element>
+    <element id="Consent.extension:comment.value[x]">
+      <path value="Consent.extension.value[x]" />
+      <short value="Comment" />
+      <definition value="The comment may include further information such as the reason why a treatment directive has been drawn up, persons who assisted the patient in agreeing the treatment directive but who are not otherwise involved in the agreement as a representative, etc." />
+      <alias value="Toelichting" />
+      <mapping>
+        <identity value="zib-treatmentdirective2-v1.0-2020EN" />
+        <map value="NL-CM:2.2.13" />
+        <comment value="Comment" />
+      </mapping>
+    </element>
+    <element id="Consent.extension:additionalAdvanceDirective">
+      <path value="Consent.extension" />
+      <sliceName value="additionalAdvanceDirective" />
+      <min value="0" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-TreatmentDirective2.AdvanceDirective" />
+      </type>
+      <condition value="hcim-treatmentdirective2-1" />

```

Comment:
Shouldn't this be replaced by 'zib-TreatmentDirective2-01'? Also see line 171.

Reply:
No: https://informatiestandaarden.nictiz.nl/wiki/FHIR:V1.0_FHIR_Profiling_Guidelines_R4#Miscellaneous

Reply:
Fixed!